src/main/java/org/alfresco/rest/framework/jacksonextensions/JacksonHelper.java

```diff
@@ -82,9 +82,8 @@ public void afterPropertiesSet() throws Exception
         //Configure the objectMapper ready for use
         objectMapper = new ObjectMapper();
         objectMapper.registerModule(module);
-        objectMapper.setDefaultPropertyInclusion(JsonInclude.Include.NON_EMPTY);
-        objectMapper.configOverride(java.util.Map.class)
-                        .setInclude(JsonInclude.Value.construct(JsonInclude.Include.NON_EMPTY, null));

```

Comment:
`JsonInclude.Value.construct(JsonInclude.Include.NON_EMPTY, null)` is equivalent to `JsonInclude.Include.NON_EMPTY, JsonInclude.Include.ALWAYS)` - not setting the inclusion criteria for `value` or `content` defaults to `JsonInclude.Include.ALWAYS`.